text/0249-v2-experiments.md

```diff
@@ -6,10 +6,12 @@ rfc pr: https://github.com/aws/aws-cdk-rfcs/pull/250
 
```

Comment:
Like all RFCs, Can we add a "working backwards" section as the first section of the RFC. Write it as 1-2 paragraphs which will be included as part of the v2.0 announcement. 

See my comment below for proposed phrasing.

Reply:
Just to clarify. We want to add a working backward section to the general RFC or specifically to option 6 adde in this PR?